Vueo: add theme

henriqueiamarino opened the pull request:
Vueo presents a portfolio theme that offers a distinctive perspective on your gallery entrances. Alternating the aspect ratio of the query loops crafts a visually stunning pattern for an enhanced viewing experience.

[Demo site](https://vueo.mystagingwebsite.com/) | [Site Content Export](https://github.com/user-attachments/files/17717947/vueo.WordPress.2024-11-12.xml.zip)

![screenshot](https://github.com/user-attachments/assets/4b68a713-edd7-4b49-9e23-28e9b6f726f9)

henriqueiamarino committed:
Vueo: add theme

iamtakashi commented:
@henriqueiamarino I love the aesthetic and the design details. Great work, as always. 🌟 

The theme is solid, too. I only noted a few minor things and suggestions. 

It's just a suggestion, but I wonder if the content width could be `842px` instead of `900px` so the content lines up with the grid in the header. Feel free to take or dismiss it.
![CleanShot 2024-11-15 at 16 59 51@2x](https://github.com/user-attachments/assets/f059610d-4960-4a24-b42e-a86c4116f7f7)
![CleanShot 2024-11-15 at 17 01 59@2x](https://github.com/user-attachments/assets/ac03d8f5-d137-424c-ac22-4a9db0253615)

It's also a suggestion, but I wonder if the background colour of a search could also be `primary` so it's the same as the button colour. However, the magnified glass icon will be a bit off-centre. Feel free to take or dismiss it.
![CleanShot 2024-11-15 at 17 25 44@2x](https://github.com/user-attachments/assets/255635e9-e5fb-400c-8b18-d653f9eb23ce)

![CleanShot 2024-11-15 at 17 28 55@2x](https://github.com/user-attachments/assets/af4c3071-ffb2-46aa-b201-a26a7424ec62)

I think that size is meant to be `1.5rem`. 
![CleanShot 2024-11-15 at 18 25 14@2x](https://github.com/user-attachments/assets/be37bda2-5ab8-4b73-9a29-ec95e5e0a304) 
![CleanShot 2024-11-15 at 18 27 15@2x](https://github.com/user-attachments/assets/4816bdba-a797-467c-8f98-cd90d85d94df)

I don't see the post meta template part `post-meta.html` has been used. Let's remove the file and the mention in the JSON file.

This is a bug in CBT, but let's make the word "search" in the search block internationalised. It's in `404.php` and `search.php`. Replace the block with this in VS Code.
```
<!-- wp:search {"label":"<?php esc_html_e('Search', 'vueo');?>","showLabel":false,"placeholder":" <?php esc_html_e('Search', 'vueo');?>","width":100,"widthUnit":"%","buttonText":"<?php esc_html_e('Search', 'vueo');?>","buttonPosition":"button-inside","buttonUseIcon":true} /-->
``` 

It's not a problem because they are meant to change, but for the social links in the `footer-pages.php`, we could use either `#` or the URL (including `https://`) of each service.
![CleanShot 2024-11-15 at 18 53 37@2x](https://github.com/user-attachments/assets/7cb050e8-d639-4e4d-9de2-0d6d68b0b4c2)

These two patterns, `front-page.php` and `post-meta.php` are not used. Let's remove them.

Since 6.7 has been released, let's change the `Tested up to` in `readme.txt` and `style.css`.

henriqueiamarino committed:
Vueo: fixes issues commented on the review

henriqueiamarino commented:
Thanks, @iamtakashi. I reviewed and fixed the following observations:

- Changed the content-width;
- Fixed the Large font preset;
- Replace the Search background color with Primary;
- Deleted the post-meta template and fixed theme.json;
- Deleted front-page.php and post-meta.php;
- Updated 'Tested up' version on style.css and readme.txt.

iamtakashi committed:
Merge remote-tracking branch 'origin/trunk' into add/vueo

iamtakashi committed:
Merge branch 'add/vueo' of github.com:Automattic/themes into add/vueo

iamtakashi committed:
Make search block text internationalised on 404 template

iamtakashi committed:
Removed URL from social link block in page footer pattern

iamtakashi commented:
@henriqueiamarino I've pushed a few commits to fix overlooked things [1](https://github.com/Automattic/themes/pull/8410/commits/1c8b879a679de539b6d580cc3bd7dd8711d161e0), [2](https://github.com/Automattic/themes/pull/8410/commits/fcfd8db159f3b436f8dacf30cee51e1937578971), but it's ready to ship!
